plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizard.java

```diff
@@ -0,0 +1,60 @@
+
+package com.google.cloud.tools.eclipse.appengine.newproject.maven;
+
+import com.google.cloud.tools.eclipse.appengine.newproject.AppEngineStandardWizardPage;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jface.operation.IRunnableWithProgress;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.ui.INewWizard;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.statushandlers.StatusManager;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class MavenArchetypeProjectWizard extends Wizard implements INewWizard {
+  private AppEngineStandardWizardPage page;
+  private AppEngineStandardProjectMavenConfig config = new AppEngineStandardProjectMavenConfig();
+
+  public MavenArchetypeProjectWizard() {
+    this.setWindowTitle("New Maven-based App Engine Standard Project");
+    page = new AppEngineStandardWizardPage(config);
+    setNeedsProgressMonitor(true);
+  }
+
+  @Override
+  public void addPages() {
+    this.addPage(page);
+  }
+
+  @Override
+  public boolean performFinish() {
+    IRunnableWithProgress runnable = new IRunnableWithProgress() {
+      @Override
+      public void run(IProgressMonitor monitor)
+          throws InvocationTargetException, InterruptedException {
+        new CreateAppEngineStandardMavenProject(config).run(monitor);
```

Comment:
This class is almost identical to the other wizard. We should at least add a todo to refactor to share code.